src/components/KeyboardToolbar/index.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { StyleSheet, Text, View, useColorScheme } from "react-native";
+
+import {
+  FocusedInputEvents,
+  KeyboardStickyView,
+} from "react-native-keyboard-controller";
+
+import { KeyboardController } from "../../bindings";
+
+import Arrow from "./Arrow";
+import Button from "./Button";
+import { colors } from "./colors";
+
+import type { LayoutChangeEvent, TextStyle, ViewStyle } from "react-native";
+
+export type KeyboardToolbarProps = {
+  Content: () => JSX.Element;
+};
+
+// TODO: accessibility
+
+const dismissKeyboard = () => KeyboardController.dismiss();
+const goToNextField = () => KeyboardController.setFocusTo("next");
+const goToPrevField = () => KeyboardController.setFocusTo("prev");
+/**
+ * `KeyboardToolbar` is a component that is shown above the keyboard with `Prev`/`Next` and
+ * `Done` buttons.
+ */
+const KeyboardToolbar: React.FC<KeyboardToolbarProps> = ({ Content }) => {
+  const theme = useColorScheme() || "light";
+  const [height, setHeight] = useState(0);
+  const [inputs, setInputs] = useState({
+    current: 0,
+    count: 0,
+  });
+  const background: ViewStyle = { backgroundColor: colors[theme].background };
+  const done: TextStyle = { color: colors[theme].primary };
+  const isPrevDisabled = inputs.current === 0;
+  const isNextDisabled = inputs.current === inputs.count - 1;
+
+  const onLayout = useCallback((e: LayoutChangeEvent) => {
+    setHeight(e.nativeEvent.layout.height);
+  }, []);
+
+  useEffect(() => {
+    const subscription = FocusedInputEvents.addListener("focusDidSet", (e) => {
+      setInputs(e);
+    });
+
+    return () => subscription.remove();
+  }, []);
+
+  return (
+    <KeyboardStickyView offset={{ closed: height }}>
+      <View
+        // TODO: check whether it's readable or not
+        accessibilityLabel="Toolbar"
+        onLayout={onLayout}
+        style={[
+          styles.toolbar,
+          height === 0 ? { marginBottom: -9999 } : null,
+          background,
+        ]}
+        testID="keyboard.toolbar"
+      >
+        <Button
+          accessibilityLabel="Previous"
+          accessibilityHint="Will move focus to previous field"
+          disabled={isPrevDisabled}
+          onPress={goToPrevField}
+          testID="keyboard.toolbar.previous"
+        >
+          <Arrow disabled={isPrevDisabled} direction="up" />
+        </Button>
+        <Button
+          accessibilityLabel="Next"
+          accessibilityHint="Will move focus to next field"
+          disabled={isNextDisabled}
+          onPress={goToNextField}
+          testID="keyboard.toolbar.next"
+        >
+          <Arrow disabled={isNextDisabled} direction="down" />
+        </Button>
+
+        <View style={styles.flex} testID="keyboard.toolbar.content">
+          {Content ? <Content /> : null}
+        </View>
+        <Button
+          accessibilityLabel="Done"
```

Comment:
There is no any sense to add this a11y label because it's the same as text

Reply:
@IvanIhnatsiuk I know, but i just want to have a better code consistency. Let's leave it for now 😉